cypress/e2e/shared/scriptQueryBuilder.sql.test.ts

```diff
@@ -42,35 +49,41 @@ describe('Script Builder', () => {
   }
 
   before(() => {
-    const writeData: string[] = []
-    for (let i = 0; i < 30; i++) {
-      writeData.push(`ndbc,air_temp_degc=70_degrees station_id_${i}=${i}`)
-      writeData.push(`ndbc2,air_temp_degc=70_degrees station_id_${i}=${i}`)
+    const generateWriteData = (value: number) => {
+      return [
+        `${measurement},${tagKey}=${tagValue} ${fieldName}=${value}`,
+        `${measurement},${tagKey}=${tagValue} ${fieldName2}=${value}`,
+        `${measurement},${tagKey}=${tagValue2} ${fieldName}=${value}`,
+        `${measurement},${tagKey}=${tagValue2} ${fieldName2}=${value}`,
+      ]
     }
 
     cy.flush().then(() => {
       return cy.signin().then(() => {
         return cy.get('@org').then(({id, name}: Organization) => {
           cy.log('add mock data')
-          cy.createBucket(id, name, 'defbuck2')
-          cy.writeData(writeData, 'defbuck')
-          cy.writeData(writeData, 'defbuck2')
+          cy.createBucket(id, name, bucketName)
+          cy.createBucket(id, name, anotherBucketName)
+
+          cy.log('create time series, with change of value over time')
+          cy.writeData(generateWriteData(100), bucketName)
+          cy.wait(2000)
+          cy.writeData(generateWriteData(20), bucketName)
+          cy.writeData(generateWriteData(20), anotherBucketName)
```

Comment:
Change value with `generateWriteData()`, so the test line cannot be just smoothed away to a single datapoint.